### ISSUE TITLE
Disable Hotkey On Settings Menu

### ISSUE THREAD

**Issue by Stateford** [Stateford/Display-Lock#71]
Currently the hotkey button is enabled on the settings menu and should be temporarily disabled until the tab is switched. The key should be unregistered, and re-registered after the switch.
The current hotkey should appear in the Hotkey control box

**Comment by idietmoran** [comment]
It is. I'm thinking the hotkey should be unregistered when a user navigates to the settings page, and reregistered when the user navigates away.

Make sure you do it on the develop branch.

Line 370-374 is where the switching is detected.

Thanks!

**Comment by Arbalest007** [comment]
@idietmoran I was wondering if I could potentially get some more clarification on the hotkey. I'm a little confused on what the hotkey is as I don't see an option for it on the "Settings" Menu or on the "Windows" Menu when running the program.

**Comment by idietmoran** [comment]
Currently this feature is only enabled on the develop branch.

There should be a small box at the bottom of the window that you can click into and set a hotkey.
Currently any registered hotkey will still activate while on this window.

Check hotkey.c in dispLib for more information.

The hotkey information is stored in a struct HOTKEY which is also in a struct SETTINGS.

Check common.h for struct implementations.

**Comment by Arbalest007** [comment]
Awesome. Thanks! That clarifies a lot. I didn't see the develop branch and was on the master branch when I opened up the project in VS.

**Comment by idietmoran** [comment]
Yep, all work should be done on develop, or a branch checked out from develop.

Any tests you may want to add would be appreciated as well (although a lot of this stuff is hard to test against)

**Comment by Arbalest007** [comment]
No problem. I was still in the process of getting acquainted with the codebase as it's my first time working with a WinAPI application so not much was changed on my end yet. Thanks for the heads up!
